<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiEditGroundOverlayDialog</class>
 <widget class="QDialog" name="UiEditGroundOverlayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="name_lbl">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>m_name</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="link_lbl">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_link"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_descriptionTab">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="m_description"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Coordinates</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="m_northLabel">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>N</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="m_north">
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="m_westLabel">
               <property name="font">
                <font>
                 <underline>true</underline>
                </font>
               </property>
               <property name="text">
                <string>W</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="m_west">
               <property name="decimals">
                <number>5</number>
               </property>
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="m_eastLabel">
               <property name="font">
                <font>
                 <underline>true</underline>
                </font>
               </property>
               <property name="text">
                <string>E</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="m_east">
               <property name="decimals">
                <number>5</number>
               </property>
               <property name="minimum">
                <double>-180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>180.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="m_southLabel">
             <property name="font">
              <font>
               <underline>true</underline>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="text">
              <string>S</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="m_south">
             <property name="decimals">
              <number>5</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="m_rotationLabel">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="m_rotation">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="minimum">
              <double>-360.000000000000000</double>
             </property>
             <property name="maximum">
              <double>360.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UiEditGroundOverlayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UiEditGroundOverlayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
